docs/source/developers/python.rst

```diff
@@ -120,37 +121,37 @@ Xcode Command Line Tools (``xcode-select --install``) are sufficient.
 On Linux, for this guide, we require a minimum of gcc 4.8, or clang 3.7 or
 higher. You can check your version by running
 
-.. code-block:: console
+.. code-block::
```

Comment:
In the PR description, you mention "use of console blocks instead of shell", but this is actually removing "concole" as well. Is this because it is the default? Or because the sphinx-copybutton will ensure that the $ is not copy-able?

Reply:
Hm, I think this is a mistake on my side (leftover before I set the default). Will delete and build the docs again to be sure.